Assets/ZSSRichTextEditor.js

```diff
@@ -346,6 +329,29 @@ ZSSEditor.getYCaretInfo = function() {
     return this.caretInfo;
 };
 
+/**
+ *  @brief      Whenever this method is called, a check will be performed on the caret position
+ *              to figure out if it needs to be wrapped in a paragraph node.
+ */
```

Comment:
Could we describe in a little more detail what kind of check we are performing in this function?